Makefile

```diff
@@ -30,6 +30,10 @@ build-final-client: move-client
 # Full build of the client from the openapi schema. Use this whenever the openapi schema is updated
 build-from-schema: install api-schema build-final-client fix-linting build
 
+# Fast build of the client from the openapi schema. Used for development purposes when installing
+# dependencies and linting isn't necessary. Currently used on dashboard to speed regeneration of models.
+development-build-from-schema: api-schema build-final-client build
```

Comment:
What's 'final'?

Reply:
On the `build-final-client`? Is the process that builds the client from the autogenerated one on the base generator. At this point, it's a direct move.

Reply:
For the sake of understanding this, is there a non-final client?

Reply:
The client by the base generator called `luneClient`. This is a bit of remnant of when we had `sed` operations so the base client was very different from the final client. I can reword this, but I believe this should be another PR 👍